### PR TITLE
[FEAT] Add examples using the browser npm package

### PR DESCRIPTION
**Pull request by tbouffard**
Add the following examples
- getting started: demonstrate the basic usage of the lib in an HTML page. Minimum code required
- local BPMN diagram load: demonstrate how to bridge an HTML file input with the lib

The lib is loaded from `unpkg`. The `getting started` example mentions `jsdelivr` as well.

Covers https://github.com/process-analytics/bpmn-visualization-js/issues/599

### CODE REVIEW

File: examples/load-local-bpmn-diagrams/index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    </section>
+</header>
+<section class="container col-10 flex-centered mt-2">
+        <section class="col-12 mt-2 relative">
+            <h2>Load local BPMN diagrams</h2>
+            <div id="file-selector">
+                <input type="file" id="btn-open-file" name="btn-open-file" class="hidden"/>
+                <label for="btn-open-file" class="btn btn-primary has-icon-right"><span>Choose a local file </span><span class="icon icon-upload mb-1"></span></label>
+            </div>
+<!--            TODO position at the right of the button-->
```
Comment (tbouffard) [inline examples/load-local-bpmn-diagrams/index.html:71]: @aibcmars can you help on this please? If we implement this change, the preview image must be updated

File: examples/load-local-bpmn-diagrams/index.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            </ul>
+        </div>
+    </section>
+</header>
+<section class="container col-10 flex-centered mt-2">
+        <section class="col-12 mt-2 relative">
+            <h2>Load local BPMN diagrams</h2>
+            <div id="file-selector">
+                <input type="file" id="btn-open-file" name="btn-open-file" class="hidden"/>
+                <label for="btn-open-file" class="btn btn-primary has-icon-right"><span>Choose a local file </span><span class="icon icon-upload mb-1"></span></label>
```
Comment (tbouffard) [inline examples/load-local-bpmn-diagrams/index.html:69]: :question: If someone find a better icon, feel free to commit

File: examples/load-local-bpmn-diagrams/index.html (hunk 1 above)
Comment (tbouffard) [reply examples/load-local-bpmn-diagrams/index.html:79]: done together, thanks

**Review by aibcmars — APPROVED** [review]
LGTM

**Review by csouchet — APPROVED** [review]

**Pull request merged**